src/Raven.Server/Documents/Handlers/RevisionsBinCleanerHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+using Raven.Server.Documents.Handlers.Processors.Revisions;
+using Raven.Server.Routing;
+
+namespace Raven.Server.Documents.Handlers;
+public class RevisionsBinCleanerHandler : DatabaseRequestHandler
+{
+    [RavenAction("/databases/*/revisions-bin-cleaner/config", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
```

Comment:
ho about changing this to `/databases/*/revisions/cleaner/config` or  `/databases/*/revisions/bin-cleaner/config` ?

Reply:
Done